JCR-1154: Throw DataStoreException instead of IOException

git-svn-id: https://svn.apache.org/repos/asf/jackrabbit/trunk@591286 13f79535-47bb-0310-9956-ffa450edef68

+import org.apache.jackrabbit.core.data.DataStoreException;
-import java.io.IOException;
-        } catch (IOException e) {
+        } catch (DataStoreException e) {
-        try {
-            return getDataRecord().getStream();
-        } catch (IOException e) {
-            throw new RepositoryException(e);
-        }
+        return getDataRecord().getStream();
-    static BLOBInDataStore getInstance(DataStore store, InputStream in) throws IOException {
+    static BLOBInDataStore getInstance(DataStore store, InputStream in) throws DataStoreException {
-    private DataRecord getDataRecord() throws IOException {
+    private DataRecord getDataRecord() throws DataStoreException {

MOV26 UPD40 MOV8 UPD43 UPD43 UPD42 UPD42 UPD43 UPD42 DEL42 DEL43 DEL42 DEL44 DEL42 DEL43 DEL42 DEL14 DEL53 DEL8 DEL12 DEL54 DEL8